Formula/rsc_2fa.rb

```diff
@@ -0,0 +1,28 @@
+# typed: false
+# frozen_string_literal: true
+
+# Homebrew formula for 2fa
```

Comment:
```suggestion
```

Reply:
Thank you the linter wanted those lines.